.github/workflows/run-tests.yml

```diff
@@ -40,7 +37,7 @@ jobs:
 
       - name: Install dependencies
         run: |
-          composer require "laravel/framework:${{ matrix.laravel }}" "orchestra/testbench:${{ matrix.testbench }}" --no-interaction --no-update
+          composer require "laravel/framework:${{ matrix.laravel }}" --dev --no-interaction --no-update
```

Comment:
Testbench is already installed with `^7.4` as the minimum version matching Laravel Framework 9.7. Requiring it here messes with Composer on Windows.

Reply:
Unless you are testing multiple versions of Laravel, this line isn't even necessary.